.github/workflows/checks.yaml

```diff
@@ -0,0 +1,26 @@
+name: Checks
+on:
+  pull_request:
+    branches:
+      - master
+jobs:
+  count_commits:
+    name: Check that there is only one commit
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout source
+      uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+        fetch-depth: 0
+    - name: Count commits
+      run: |
+        count=`git rev-list --count ${{ github.event.pull_request.base.sha }}..${{ github.event.pull_request.head.sha }}`
+        echo Number of commits: $count
+        git log --oneline ${{ github.event.pull_request.base.sha }}..${{ github.event.pull_request.head.sha }}
```

Comment:
I think this line does not print anything. Maybe the arguments need to be reversed?

Reply:
Scratch that, I am mistaken.

Reply:
You mean the last line?

No, you were correct in that it didn't print all the commits, so fixed that, but arguments should not be reversed.

Reply:
Now it prints 2 lines.

Reply:
Fixed.